### PR TITLE
build: handle firefox development

### PR DESCRIPTION
**Pull request by felix-lambert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +80,15 @@ yarn start
 > Chrom(e|ium) browser is recommend for development.
 > see https://developer.chrome.com/extensions/getstarted#manifest
 
+### For firefox development extension
+
+```bash
+npm install --global web-ext
```
Comment (lutangar) [inline README.md:86]: This part is not required since `web-ext` is already a dependency.

File: package.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
     "start": "webpack --watch --progress --mode=development",
     "start:profiles": "webpack-dev-server --progress --mode=development --config webpack.profiles.config.js --inline",
     "start:firefox": "webpack --watch --progress --mode=development --env.PLATFORM=firefox",
+    "run:firefox": "cd build/development/firefox && web-ext run",
```
Comment (lutangar) [inline package.json:12]: Prefer to run it via `start:firefox` in dev, since it uses webpack watch mode.

File: README.md [hunk 3]
```diff
@@ -78,6 +80,15 @@ yarn start
...
+yarn start:firefox
+yarn run:firefox
```
Comment (lutangar) [inline README.md:88]:
```
yarn start:firefox
```
is sufficient.

**Review by lutangar — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (felix-lambert) [reply package.json:12]: It's a command that only runs the navigator with the extension installed. You think we don't need this command?

Comment (lutangar) [reply package.json]: Oh I see! This is actually interesting let's keep it.

Comment (felix-lambert) [reply package.json:12]: Will check if webpack watch mode works with this command

**Review by lutangar — APPROVED** [review]

**Pull request closed without merging**